Gifski/MainWindowController.swift

```diff
@@ -61,6 +62,10 @@ final class MainWindowController: NSWindowController {
 		}
 	}
 
+	var conversionCompleted: Bool {
+		return conversionCompletedView.isHidden == false && self.outUrl != nil
```

Comment:
ah, just noticed, the `self` is probably not needed here

Reply:
```suggestion
		return conversionCompletedView.isHidden == false && outUrl != nil
```

Reply:
Oh nice I can actually make a suggestion to my own PR and commit it without going to the branch and pushing to upstream. So cool!